mldsa/sign.c

```diff
@@ -96,14 +96,37 @@ int crypto_sign_keypair_internal(uint8_t *pk, uint8_t *sk,
   /* Compute H(rho, t1) and write secret key */
   shake256(tr, MLDSA_TRBYTES, pk, CRYPTO_PUBLICKEYBYTES);
   pack_sk(sk, rho, tr, key, &t0, &s1, &s2);
+
+  /* FIPS 204. Section 3.6.3 Destruction of intermediate values.
+   * Note:
+   * - "The seed ξ can be stored for later expansion using
+   * ML-DSA.KeyGen_internal"
+   * - "The matrix Â can be stored so that it need not be recomputed" */
```

Comment:
Not sure I follow this. Currently we do not keep those around, so they should be zeroized? 
If we do want to keep it around, we need to change the API first. 
Please zeroize everything for now.